### DISCUSSION TITLE
好心人请带上开发商名字啊，防止后面的人进坑

### DISCUSSION THREAD

**Discussion by ZjieHU** [WeNeedHome/SummaryOfLoanSuspension#677] [Ideas]
作为马上要买房的业主，看到这个帖子很心慌，一方面深深同情大家的遭遇，另一方面也很害怕碰到这样的事情。
哪个好心人可以贴出来烂尾的开发商名字，好让后面人避坑啊，🙏

**Comment by C-Joey** [comment]
你现在不可能买期房吧

**Comment by 1nter5e0** [comment]
不知道后面哪位好心人有空整理下

**Comment by TALK1949** [comment]
我建议这几年最好不要考虑买房子，先观察一阵子……
当然，刚需的话建议买建好的。
可以考虑二手房。

**Comment by killunix** [comment]
现在还买期房？纯粹是找死！
